aks/secrets/README.md

```diff
@@ -0,0 +1,28 @@
+# AKS Secrets
+
+Terraform code for extracting secrets from Azure Key Vaults.
+
+## Usage
+
+```terraform
+module "infrastructure_secrets" {
+  source = "git::https://github.com/DFE-Digital/terraform-modules.git//aks/secrets?ref=stable"
+
+  azure_resource_prefix = "s189"
+  service_short         = "afqts"
+  config_short          = "dv"
+  name                  = "infrastructure"
```

Comment:
As discussed, we should use "inf" so it would be nice to reflect this in the example

Reply:
I've amended the commit!